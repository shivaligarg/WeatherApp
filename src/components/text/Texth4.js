import React from 'react';
import { Text} from 'react-native-elements';
export const Texth4 = (props) =>{
    const {style,title} =props
    return (title?<Text h4 h4Style={style}>{title}</Text>:null);
}