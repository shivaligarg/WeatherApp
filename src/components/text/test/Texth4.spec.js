import React from "react";
import {shallow} from 'enzyme';
import { Texth4 } from "../Texth4";
import { Text } from "react-native-elements";
describe('Texth4 Test cases --------',()=>{
    let wrapper;
    beforeEach(()=>{
        wrapper=shallow(<Texth4 title={'Settings'} style={{color:'black'}}/>);
    });
    it("Texth4 Snapshot match",()=>{
        expect(wrapper).toMatchSnapshot();
    });
    it("Texth4 Lost View Snapshot",()=>{
        wrapper.setProps({title :''});
        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find(Text).length).toBe(0);
    });
});