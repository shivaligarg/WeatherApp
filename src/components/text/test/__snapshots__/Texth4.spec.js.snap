// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Texth4 Test cases -------- Texth4 Lost View Snapshot 1`] = `ShallowWrapper {}`;

exports[`Texth4 Test cases -------- Texth4 Snapshot match 1`] = `ShallowWrapper {}`;
